lib/node_modules/@stdlib/stats/base/dists/uniform/cdf/benchmark/benchmark.js

```diff
@@ -32,16 +33,24 @@ var cdf = require( './../lib' );
 bench( pkg, function benchmark( b ) {
 	var min;
 	var max;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array(len);
+	min = new Float64Array(len);
+	max = new Float64Array(len);
+	for(i = 0; i < len; i++){
+		x[ i ] = uniform(-10.0, 20.0);
+		min[ i ] = uniform(-20.0, 20.0);
+		max[ i ] = min[ i ] + uniform(0.0, 40.0);
+	}
+
 	b.tic();
 	for ( i = 0; i < b.iterations; i++ ) {
-		x = ( randu() * 20.0 ) - 10.0;
-		min = ( randu() * 20.0 ) - 20.0;
-		max = min + ( randu() * 40.0 );
-		y = cdf( x, min, max );
+		y = cdf( x[ i % len], min[ i % len], max[ i % len] );
```

Comment:
```suggestion
		y = cdf( x[ i % len ], min[ i % len ], max[ i % len ] );
```

This applies throughout the PR